shoop/admin/modules/orders/static_src/create/reducers/lines.js

```diff
@@ -111,33 +124,44 @@ function setLineProperty(state, {payload}) {
                 updates.type = value;
                 break;
             case "quantity":
-                var quantity = Math.max(0, parseFloat(value));
+                const quantity = Math.max(0, ensureNumericValue(value, 1));
```

Comment:
Actually if this is product line quantity should default as min quantity defined for product. That value should got with retrieveProductData.